packages/snap-preact-components/src/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -13,6 +14,8 @@ import { Swiper, SwiperSlide } from 'swiper/react';
 import { defined } from '../../../utilities';
 import { Theme, useTheme, CacheProvider } from '../../../providers';
 import { ComponentProps } from '../../../types';
+import { useDisplaySettings } from '../../../hooks/useDisplaySettings';
+import { BreakpointsProps } from '../../Organisms/Results';
```

Comment:
Since `BreakpointsProps` is pretty generic and also being used in these two files, can you move the typing from the results component to the types.ts file?